src/main/java/org/datacleaner/extension/elasticsearch/ElasticSearchFullSearchTransformer.java

```diff
@@ -76,6 +87,11 @@
     @Configured(order = 4, required = false)
     String searchFieldName;
     
+    @Configured(order = 5, required = true)
+    @Description("Result sizes above 1 only available for output data stream")
+    Integer getNumberOfResults = 1;
```

Comment:
This should still change to `maximumNumberOfResults` (or, if you don't want to change it, then at least change line 111 to use the property name "Get number of results".